server/util/api/users.js

```diff
@@ -113,6 +116,8 @@ async function userInfoQuery({
   wallet,
   email,
   firebaseUserId,
+  platform,
+  platformUserId,
```

Comment:
changes on `userInfoQuery()` should be reflected in `checkUserInfoUniqueness()` and `checkIsOldUser()`